packages/core/src/Stores/ui-store.js

```diff
@@ -602,6 +602,7 @@ export default class UIStore extends BaseStore {
     @action.bound
     setCurrentFocus(value) {
         this.current_focus = value;
+        this.is_keyboard_active = this.current_focus !== null && this.is_mobile && isTouchDevice();
```

Comment:
Can we rename it to `is_onscreen_keyboard_active` to make it more specific?
Also, if this is an `observed` state, should we have a default `value` set in the store?

Reply:
update the name 
we have default value(line 55) I've used this observable on dbot-self-exclusion